Duplicati.Library.Backend.Tardigrade/TardigradeBackend.cs

```diff
@@ -0,0 +1,330 @@
+﻿using Duplicati.Library.Interface;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using uplink.NET.Interfaces;
+using uplink.NET.Models;
+using uplink.NET.Services;
+
+namespace Duplicati.Library.Backend.Tardigrade
+{
+    public class Tardigrade : IBackend, IStreamingBackend
```

Comment:
`IStreamingBackend` implements `IBackend`.  Is implementing `IBackend` here redundant?

Reply:
You're Right, I've educed it to IStreamingBackend